client/my-sites/stats/stats-period-navigation/index.jsx

```diff
@@ -344,5 +359,6 @@ export default flowRight(
 	connectComponent,
 	localize,
 	withRtl,
-	withLocalizedMoment
+	withLocalizedMoment,
+	withStatsPurchases
```

Comment:
@Automattic/red it seems to be a nice way to use hooks for class component 🥳

Reply:
Also wrap a selector is even nicer:

https://github.com/Automattic/wp-calypso/pull/89380/files#diff-fc24058576fcc9d1fe4a8859e1d0a7ea6cb6c027624e0f1bc5e209b96ddb13a1R272

We could either use it directly in class and functional components, and it could easily used in hooks too! Dotcom folks are clever.